### PR TITLE
Initial version: Lambda function and create-job runner

### PR DESCRIPTION
**Pull request by arielshaqed**
Tested by semi-manually creating a manifest and running it.  Then breaking one file on that manifest (setting a nonexistent dstBucket) and seeing that exactly one error is reported in the report log.

### CODE REVIEW

File: run_copy.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        'Spec': {
+          'Format': 'S3BatchOperations_CSV_20180820',
+          'Fields': ['Bucket', 'Key'],
+        },
+        'Location': {
+          'ObjectArn': 'arn:aws:s3:::%s/%s' % (getBucket(csv_path), getPath(csv_path)),
+          'ETag': '<none>'
+        }
+      },
+      # Description='renaming distcp on %s' % csv_path,
```
Comment (ozkatz) [inline run_copy.py:56]: remove commented-out code..

**Comment by arielshaqed** [comment]
Thanks!  I think I did everything, PTAL.

File: fake-manifest-from-list.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import sys
+
+
+srcBucket, dstBucket = sys.argv[1:3]
+if srcBucket is None:
+  raise 'Need source bucket name arg'
+if dstBucket is None:
+  raise 'Need destination bucket name arg'
+
+srcBucketArn = 'arn:aws:s3:::%s' % srcBucket
```
Comment (ozkatz) [inline fake-manifest-from-list.py:21]: PEP8 here as well? indentation and camelCasing

File: lambda_handler.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  srcBucket = s3BucketArn.split(':::')[-1]
+
+  try:
+    # Must set a result later
+    resultCode = None
+    resultString = ''
+
+    # Decode the JSON string that was encoded into the S3 Key value and convert the
+    # resulting string into a JSON structure.
+    s3Key_decoded = unquote_plus(s3Key)
```
Comment (ozkatz) [inline lambda_handler.py:67]: cameCasing_with_undescores? also, 2-space indents..

File: run_copy.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+import os
+import re
+import click
+import json
+import boto3
+from botocore.exceptions import ClientError
+
+
+def parse_S3(s3_url):
```
Comment (ozkatz) [inline run_copy.py:13]: I'd stick with either lowercase s3 or capitalised, but keep it consistent..

Comment (arielshaqed) [reply run_copy.py]: pycodestyle allows it, and I did have some method to the names, but I guess "`s3`" throughout will be easiest.

**Comment by arielshaqed** [comment]
@ozkatz I added some changes and finished an end-to-end run.  PTAL.

**Comment by arielshaqed** [comment]
Thanks!
Pulling. I'd like to publish it together with lakeFS, I'll copy PRs from there to prepare it on Sunday.

**Pull request merged**